### PR TITLE
SPOI-4443: Adding LZO support in contrib

### PR DESCRIPTION
**Pull request by DT-Priyanka**

### CODE REVIEW

File: contrib/src/main/java/com/datatorrent/contrib/lib/io/FilterStreamCodec.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import java.io.IOException;
+import java.io.OutputStream;
+
+import org.anarres.lzo.LzoAlgorithm;
+import org.anarres.lzo.LzoConstraint;
+import org.anarres.lzo.LzoLibrary;
+import org.anarres.lzo.LzopOutputStream;
+
+import com.datatorrent.lib.io.fs.FilterStreamContext;
+import com.datatorrent.lib.io.fs.FilterStreamProvider;
```
Comment (PramodSSImmaneni) [inline contrib/src/main/java/com/datatorrent/contrib/lib/io/FilterStreamCodec.java]: Add a space

File: contrib/pom.xml [hunk 2]
```diff
@@ -659,5 +659,10 @@
       <version>1.0</version>
       <optional>true</optional>
     </dependency>
+    <dependency>
+	  <groupId>org.anarres.lzo</groupId>
+	  <artifactId>lzo-core</artifactId>
+	  <version>1.0.4</version>
```
Comment (PramodSSImmaneni) [inline contrib/pom.xml]: add optional

File: contrib/src/main/java/com/datatorrent/contrib/lib/io/FilterStreamCodec.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+public class FilterStreamCodec
```
Comment (PramodSSImmaneni) [inline contrib/src/main/java/com/datatorrent/contrib/lib/io/FilterStreamCodec.java]: How about using a different name to distinguish it from the other FilterStreamCodec

**Comment by 243826** [comment]
This pull request needs to be backed out. We cannot use GPL software as dependency.

Priyanka/Pramod - please treat this urgent and back out these changes.

**Pull request merged**